treeherder/config/settings.py

```diff
@@ -34,8 +35,18 @@
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = env("TREEHERDER_DJANGO_SECRET_KEY", default='secret-key-of-at-least-50-characters-to-pass-check-deploy')
 
+# Delete the Pulse automatically when no consumers left
+PULSE_AUTO_DELETE_QUEUES = env.bool("PULSE_AUTO_DELETE_QUEUES", default=False)
+
+# Changing PULSE_AUTO_DELETE_QUEUES to true when treeherder is running in virtual environment
+if hasattr(os.environ, 'VIRTUAL_ENV'):
+    PULSE_AUTO_DELETE_QUEUES = True
+
 # Hosts
 SITE_URL = env("SITE_URL", default='http://localhost:8000')
+if env("HEROKU_REVIEW_APP", default=False):
```

Comment:
Could you please delete this?
It's already defined in [here](https://github.com/mozilla/treeherder/blob/8fbda2feecc0c8bdd3e1bb2a43c15a7e479261c8/treeherder/config/settings.py#L108).

Thanks!